templates/factory.template

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Generated factory for %class%
+ */
+
+%namespace%
+use Psr\Container\ContainerInterface;
+use Zend\Di\CodeGenerator\FactoryInterface;
+%use_array_key_exists%
```

Comment:
We generally separate `use` statements with an extra newline between contextual groups (classes/interfaces, functions, constants).

Reply:
Test covered, see: https://github.com/zendframework/zend-di/pull/38/files#diff-0a9dd3a0615cada8b15d81c973d3ce00R9